src/charts/stacked-area.js

```diff
@@ -941,18 +962,22 @@ define(function(require){
 
             values.forEach((d, index) => {
                 let marker = verticalMarkerContainer
-                              .append('g')
-                                .classed('circle-container', true)
-                                  .append('circle')
-                                    .classed('data-point-highlighter', true)
-                                    .attr('cx', highlightCircleSize)
-                                    .attr('cy', 0)
-                                    .attr('r', highlightCircleRadius)
-                                    .style('stroke-width', highlightCircleStroke)
-                                    .style('stroke', categoryColorMap[d.name])
-                                    .on('touchstart click', function() {
```

Comment:
will removing the `touchstart` event affect functionality for mobile users?

Reply:
I don't think so, it was a security thing but I don't think is necessary anymore. 
Will check though